x-pack/test/detection_engine_api_integration/security_and_spaces/roles_users_utils/index.ts

```diff
@@ -26,10 +27,10 @@ import {
 } from '../../../../plugins/security_solution/server/lib/detection_engine/scripts/roles_users';
 
 import { ROLES } from '../../../../plugins/security_solution/common/test';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { services } from '../../common/services';
 
 export const createUserAndRole = async (
-  securityService: ReturnType<FtrProviderContext['getService']>,
+  securityService: ProvidedType<typeof services['security']>,
```

Comment:
Nit: I don't see `ProvidedType` being used much externally, if at all; it seems more common to pass `getService: FtrProviderContext['getService']` to helper functions like this.

Reply:
The trick for this and why I can't use that is because we want the return type from it and it's function overloaded which does not work well for type script.

Reply:
Yeah I see what you're saying though. Most people are passing the service directly and _then_ calling the getService('security') against that and then using it.

I can change this to use that and remove the ProvidedType. Might be better for us.